src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -56,17 +58,35 @@ function EnsureGroup($group) {
 }
 
 function EnsureMember($group, $userOrGroup) {
-    if (!(Get-LocalGroupMember -Group $group -Member $userOrGroup -ErrorAction SilentlyContinue)) {
+    $modify = !(Get-LocalGroupMember -Group $group -Member $userOrGroup -ErrorAction SilentlyContinue)
+    if ($modify) {
         Add-LocalGroupMember -Group $group -Member $userOrGroup | Out-Null
     }
+    return $modify
 }
 
 ####################### Main script ################################
 
 ## TODO: Consider creating a lock around this script because it is not concurrency safe
 
 if ($install) {
-    InstallAPI
+    InstallAPI $serviceName
+} else {
+    try {
+        Invoke-WebRequest -UseDefaultCredentials -UseBasicParsing "https://localhost:$adminAPIPort" | Out-Null

```

Comment:
I decided to check endpoint first because this would enable dev scenario where services are not installed